src/components/sliders/defaults.js

```diff
@@ -73,12 +73,17 @@ function sliderDefaults(sliderIn, sliderOut, layoutOut) {
     coerce('transition.duration');
     coerce('transition.easing');
 
-    Lib.coerceFont(coerce, 'font', layoutOut.font);
-    Lib.coerceFont(coerce, 'currentvalue.font', layoutOut.font);
-
-    coerce('bgcolor', layoutOut.paper_bgcolor);
+    coerce('bgcolor');
```

Comment:
I see  the default `bgcolor` isn't inherited from `layoutOut.paper_bgcolor` anymore. Is this on purpose?